### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by Stefanwullems**

### CODE REVIEW

File: src/utils/loremIpsum.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const loremIpsum = `Lorem Ipsum is simply dummy text of the printing and typesetting industry.
```
Comment (hongaar) [inline src/utils/loremIpsum.ts]: Could also use faker:

```ts
import faker from 'faker'

const dummyText = faker.lorem.sentence(4)
```

File: src/Modal/Modal.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const Header = styled.div`
+  padding: 15px 30px;
+  color: ${fromTheme(theme => theme.colours.gray)};
+  font-size: 28px;
+  font-weight: 700;
+`
+
+const Body = styled.div`
+  padding: 30px;
+  height: 400px;
```
Comment (hongaar) [inline src/Modal/Modal.tsx]: I'd not set the height explicitly here, just keep it dynamic.

File: src/Modal/Modal.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import { fromTheme, defaultStyledProps, StyledProps } from '../utils/styled'
+import { Theme } from '../themes'
+
+const ModalWrapper = styled.div`
+  position: absolute;
+  width: 600px;
+`
+
+const Header = styled.div`
+  padding: 15px 30px;
```
Comment (hongaar) [inline src/Modal/Modal.tsx]: Please use theme's spacing for this and other paddings/margings. 

```
padding: ${fromTheme(theme => theme.global.spacing)}em ${fromTheme(theme => theme.global.spacing * 2)}em;
```

Helps prevent alignment issues like 
image

File: src/Modal/Modal.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import styled from 'styled-components'
+import { fromTheme, defaultStyledProps, StyledProps } from '../utils/styled'
+import { Theme } from '../themes'
+
+const ModalWrapper = styled.div`
+  position: absolute;
+  width: 600px;
+`
+
+const Header = styled.div`
```
Comment (hongaar) [inline src/Modal/Modal.tsx]: I'm creating a Heading component in #28, let's re-use that. I'm doing some more stuff on that branch, so if we merge this first, I'll integrate it in the Modal component.

File: src/Modal/Modal.tsx [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  height: 400px;
+  border-top: solid 1px ${fromTheme(theme => theme.colours.gray)};
+  border-bottom: solid 1px ${fromTheme(theme => theme.colours.gray)};
+`
+
+const Footer = styled.div`
+  padding: 20px 30px 20px 30px;
+  color: ${fromTheme(theme => theme.colours.gray)};
+  display: flex;
+  flex-direction: row-reverse;
```
Comment (hongaar) [inline src/Modal/Modal.tsx]: You could get rid of the ButtonWrapper by adding this to the Footer styles:

```ts
const Footer = styled.div`
  // ... existing css

  ${Button} {
    margin-left: 20px; // or fromTheme
  }
`
```

File: src/Modal/Modal.tsx [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+`
+
+const ButtonWrapper = styled.div`
+  margin-left: 20px;
+`
+
+interface ModalProps {
+  title: string
+  text: string
```
Comment (hongaar) [inline src/Modal/Modal.tsx]: Or use `children`? Not sure about this one, maybe @Michiel87 can chip in.

File: src/utils/loremIpsum.ts (hunk 1 above)
Comment (Michiel87) [reply src/utils/loremIpsum.ts]: not good for storybook snapshots

Comment (hongaar) [reply src/utils/loremIpsum.ts]: oh kak yes you're right. I'm using them all over the place. will research if it's possible to 'freeze' faker for snapshots

Comment (Michiel87) [reply src/utils/loremIpsum.ts]: cool

**Pull request merged**